Drone/drone.py

```diff
@@ -0,0 +1,18 @@
+import threading
+import logging
+
+# Ours
+from drone_base import DroneBase
+from ..Utilities import constants as c
```

Comment:
I started to make the suggested changes to drone_base.py, but things quickly became bloated. Here is what it would look like:
```
from ..Utilities.constants import CONNECTION_STRING_REAL, CONNECTION_STRING_SIMULATOR, 
HEARTBEAT_TIMEOUT, DEFAULT_ARM_TIMEOUT, GUIDED, ARM_RETRY_DELAY, DEFAULT_TAKEOFF_CUTOFF_TIME, 
DEFAULT_TAKEOFF_THRUST, PERCENT_TARGET_ALTITUDE, PERCENT_ALTITUDE_THRUST_ADJUSTMENT_THRESHOLD,
SMOOTH_TAKEOFF_THRUST, MAV_ATTITUDE_MSG_RESEND_DELAY, LAND, MAV_VELOCITY_MSG_RESEND_DELAY,
MAV_HOVER_MSG_RESEND_DELAY
```
One thing I am noticing is that many of the constants are only used in one file. Would it be a good idea to define file-specific constants at the top of each file and only put shared constants in the constants folder.

Or we could import constants (not as c) and have write out constants.SOME_VALUE for all of them.

Thoughts?